mysql-test/t/mysql_57_sys_upgrade.test

```diff
@@ -0,0 +1,56 @@
+--source include/not_valgrind.inc
+--source include/have_innodb_16k.inc
+--source include/mysql_upgrade_preparation.inc
+
+--let $MYSQLD_DATADIR= `select @@datadir`
+--echo # Set different paths for --datadir
+--let $MYSQLD_DATADIR1 = $MYSQL_TMP_DIR/data57
+
+--echo ###########################################################################
+--echo # Stop the default mtr server
+--echo ###########################################################################
+
+--echo # Stop DB server which was created by MTR default
+--let $shutdown_server_timeout = 300
+--source include/shutdown_mysqld.inc
+
+
+--echo ###########################################################################
+--echo # PS#8050: PS 8.0 upgrade (from PS 5.7) crashes with Assertion failure: dict0dict.cc:1236:table2 == nullptr
```

Comment:
use PS-XXXX format instead of PS#XXXX

Reply:
fixed. Also test is successful on local with PS- changes

xtrabackup_prepare_func - Binaries are debug compiled
Using 'all' suites
Collecting tests
Checking leftover processes
Removing old var directory
 - WARNING: Using the 'mysql-test/var' symlink
Creating var directory '/home/rahul.malik/MySQL/src/p8/bld/mysql-test/var'
Installing system database
Using parallel: 1

==============================================================================
                  TEST NAME                       RESULT  TIME (ms) COMMENT
------------------------------------------------------------------------------
[ 50%] main.mysql_57_sys_upgrade                 [ pass ]  56919
[100%] shutdown_report                           [ pass ]
------------------------------------------------------------------------------
The servers were restarted 0 times
The servers were reinitialized 0 times
Spent 56.919 of 79 seconds executing testcases

Completed: All 2 tests were successful.